egcg_core/util.py

```diff
@@ -102,6 +102,9 @@ def move_dir(src_dir, dest_dir):
 def _move_file_or_link_target(file_path, dest_dir):
     """Move a file to a destination directory"""
     dest_file = os.path.join(dest_dir, os.path.basename(file_path))
+    if os.path.isdir(file_path):
+        app_logger.warning('Attempting to move a dir with _move_file_or_link_target')
+        return
```

Comment:
It occurs to me that if `move_dir` works properly, this `os.path.isdir` will never be true because `move_dir` would have recurred instead. Fairly minor, though.